src/main/java/org/broadinstitute/dsm/db/DDPInstance.java

```diff
@@ -267,7 +276,7 @@ public static boolean getRole(@NonNull String realm, @NonNull String role) {
         return (boolean) results.resultValue;
     }
 
-    private static DDPInstance getDDPInstanceWithRoleFormResultSet(@NonNull ResultSet rs) throws SQLException {
+    public static DDPInstance getDDPInstanceWithRoleFormResultSet(@NonNull ResultSet rs) throws SQLException {
```

Comment:
Passing in a `ResultSet` makes it hard to see what the method is really operating on (and can also make things hard to test).  Can you pull out the values you need from the resultset and create arguments for each?

Reply:
Agreed! I'll fix this

Reply:
On second thoughts, this code is used a lot, I will make a ticket for this change but I'd rather not to do this big change in this ticket.